lib/common/xml_logging.c

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright 2023 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
```

Comment:
"Display" sounds to me like we're displaying to a screen rather than (often) sending it to a log file. It beats sounding log-specific or adding confusion with `output_xml.c` though, so I'll probably use that for now.

Reply:
Yeah display or show was the best I could come up with for previous cases of things that can log or output